resources/user-agents/browsers/iron-mobile/iron-mobile-generic.json

```diff
@@ -24,69 +24,16 @@
       },
       "children": [
         {
-          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Chrome/* MobileIron/* Mobile Safari/*",
+          "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*) Chrome/* Iron* Safari/*",
           "platforms": [
             "Android_7_1", "Android_7_0",
-            "Android_6_1", "Android_6_0",
```

Comment:
Apparently there was no Android 6.1 version: https://en.wikipedia.org/wiki/Android_version_history

So I've been starting to remove it when I encounter it.  Should probably make a PR to remove across all files at some point.